modules/persistent-storage-csi-smb-cifs-provision-static.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-provision-static_{context}"]
+= Static provisioning
+
+You can use static provisioning to create a persistent volume (PV) and persistent volume claim (PVC) to consume existing Server Message Block protocol (SMB) shares:
+
+.Prerequisites
+* Access to the {product-title} web console.
+* {FeatureName} CSI Driver Operator and driver installed.
+
+.Procedure
+
+To set up static provisioning:
+
+. Create a PV by running the following command with the following example YAML file:
++
+[source,terminal]
+----
+$ oc create -f <pv_file_name>.yaml <1>
+----
+<1> The name of the PV YAML file.
++
+.Example PV YAML file
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  annotations:
+    pv.kubernetes.io/provisioned-by: smb.csi.k8s.io
+  name: <pv_name> <1>
+spec:
+  capacity:
+    storage: 100Gi
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  mountOptions:
+    - dir_mode=0777
+    - file_mode=0777
+  csi:
+    driver: smb.csi.k8s.io
+    volumeHandle: smb-server.default.svc.cluster.local/share/ <2>
+    volumeAttributes:
+      source: //<hostname>/<shares> <3>
+    nodeStageSecretRef:
+      name: <secret_name_shares> <4>
+      namespace: <namespace> <5>
+----
+<1> The name of the PV.
+<2> `volumeHandle` format: {smb-server-address}#{sub-dir-name}#{share-name}. Ensure that this value is unique for every share in the cluster.
```

Comment:
Why do we have a "#" sign instead of a "."?

Reply:
Weirdly, that is what is called out in the K8s example. However, the actual YAML file content uses dot, and that is what I used when testing it, so I'll change it.